azure-cron.yml

```diff
@@ -33,11 +33,6 @@ jobs:
       - checkout: self
       - bash: ci/dev-env-install.sh
         displayName: 'Build/Install the Developer Environment'
-      - bash: ci/configure-bazel.sh
-        displayName: 'Configure Bazel'
-        env:
-          IS_FORK: $(System.PullRequest.IsFork)
-          GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
```

Comment:
AFAICT the script really only configures Bazel for pushing, and this job doesn't need to push.

Reply:
Sounds right to me :+1: